tests/Console/Command/Ci/CiRunCommandTest.php

```diff
@@ -54,13 +58,18 @@ protected function setUp(): void {
     $this->eventDispatcher
       ->dispatch(Argument::cetera())
       ->willReturn(new \stdClass());
+    $this->env = $this->prophesize(EnvFacade::class);
+    $this->env
+      ->get(Argument::any())
+      ->willReturn();
```

Comment:
I didn't know you could call `::willReturn()` without any arguments. I assume that will just make it return `NULL`. It might be good to be explicit for readability:

```suggestion
      ->willReturn(NULL);
```